test_regress/t/t_lint_eofline.pl

```diff
@@ -26,7 +26,7 @@ sub gen {
 gen($Self->{top_filename});
 
 lint(
-    verilator_flags2 => ["--lint-only -Wall -Wno-DECLFILENAME -E"],
+    verilator_flags2 => ["--no-std", "--lint-only -Wall -Wno-DECLFILENAME -E"],
```

Comment:
The flag isn't there anymore, but this is because -E implies it.

I'm not sure if I understood you correctly -- I think it does makes sense to use --no-std here (explicitly or not), as this test preprocesses a single file that doesn't produce the warning. The output is compared with a golden file. Parsing std would only add noise to it.

In contrast, t_lint_eofline_bad.pl creates a file that does produce the warning, the script doesn't use --no-std.

Reply:
I was referring to needing it explicitly, I agree -E as you now do should imply it.

With -E implying --no-std, please revert adding --no-std to several of the .pl files that use -E and as such should not need the --no-std in the .pl file.  With those changes this should be ready to merge, thanks.

Reply:
Okay, it's ready then.